liquidsoap.spec

```diff
@@ -43,60 +44,75 @@ BuildRequires: libstdc++-static
+BuildRequires: ocaml-soundtouch-devel
+BuildRequires: ocaml-speex-devel
 BuildRequires: ocaml-ssl-devel
-BuildRequires: ocaml-taglib
+BuildRequires: ocaml-taglib-devel
 BuildRequires: ocaml-theora-devel
-BuildRequires: ocaml-vorbis >= 0.7.0
-BuildRequires: ocaml-xmlm
+BuildRequires: ocaml-vorbis-devel
 BuildRequires: ocaml-xmlm-devel
-BuildRequires: ocaml-xmlplaylist
+BuildRequires: ocaml-xmlplaylist-devel
 BuildRequires: ocaml-yojson-devel
 BuildRequires: opus-devel
-BuildRequires: ocaml-pcre-devel
+BuildRequires: perl-XML-DOM
+BuildRequires: pcre-devel
 BuildRequires: soundtouch-devel
 BuildRequires: speex-devel
 BuildRequires: systemd
 BuildRequires: taglib-devel
-
+# needed to find pkg-config in my fedora containers since they do not include which
+%{?fedora:BuildRequires: which}
 Requires(pre): shadow-utils
 
-Requires: lame
-Requires: libmad
 
 %description
-Liquidsoap is a powerful and flexible language for describing your streams. It offers a rich collection of
-operators that you can combine at will, giving you more power than you need for creating or transforming
-streams. But liquidsoap is still very light and easy to use, in the Unix tradition of simple strong
+Liquidsoap is a powerful and flexible language for describing your streams. It
+offers a rich collection of operators that you can combine at will, giving you
+more power than you need for creating or transforming streams. But liquidsoap
+is still very light and easy to use, in the Unix tradition of simple strong
 components working together.
 
+
+%package     doc
+Summary:     HTML documentation for %{name}
+
+%description doc
+Extensive HTML documentation for %{name}. Contains the complete liquidsoap
+website and documentation as generated from the source plus examples.
+
+
 %prep
 %setup -q
-./configure --with-internal-glib --disable-camomile --prefix=%{_exec_prefix} --sysconfdir=/etc --mandir=/usr/share/man --localstatedir=/var --disable-ldconf
+# fix for E: incorrect-fsf-address
+grep -rl --include="*.ml" --include="*.mll" --include="*.mly" 'Temple Place' src/ | xargs -n1 sed -i -e 's@59 Temple Place, Suite 330, Boston, MA  02111-1307@51 Franklin Street, Fifth Floor, Boston, MA 02110-1301@g'
+# use _pic runtime variant if ocamlc was compiled with -fPIC
+ocamlopt -config | grep 'ocamlc_cflags:.*-fPIC' >/dev/null && export OCAMLFLAGS="-runtime-variant _pic"
+./configure --disable-camomile --prefix=%{_exec_prefix} --sysconfdir=/etc --mandir=/usr/share/man --localstatedir=/var --disable-ldconf
 
 %build
 make
+make doc
 
 %install
-%make_install%{_exec_prefix} OCAMLFIND_DESTDIR=%{buildroot}%{_exec_prefix} prefix=%{buildroot}%{_exec_prefix} sysconfdir=%{buildroot}/etc mandir=%{buildroot}%{_exec_prefix}/share/man localstatedir=%{buildroot}/var
+make install %{_exec_prefix} OCAMLFIND_DESTDIR=%{buildroot}%{_exec_prefix} prefix=%{buildroot}%{_exec_prefix} sysconfdir=%{buildroot}/etc mandir=%{buildroot}%{_exec_prefix}/share/man localstatedir=%{buildroot}/var
```

Comment:
I guess you can't use `%make_install` here for some reasons, correct? If yes, consider to add a comment explaining why the `%make_install` macro can't be used here and use the appropriate directory macros, such as:
* `sysconfdir=%{buildroot}/%{_sysconfdir}`
* `mandir=%{buildroot}%{_mandir}`
* `localstatedir=%{buildroot}/%{_localstatedir}`

Reply:
Fixed in 23e5676